lib/form_data.js

```diff
@@ -175,6 +175,8 @@ FormData.prototype._multiPartHeader = function(field, value, options) {
       options.contentType ||
       FormData.DEFAULT_CONTENT_TYPE
     );
+  } else if (options.contentType) {
```

Comment:
Each of the other parts of the "if else" chain contains a `|| FormData.DEFAULT_CONTENT_TYPE`. This should work for stream no?